### ISSUE TITLE
Canal+VOD CoreElec 9.2.8 S905x

### ISSUE THREAD

**Issue by rocky57** [mbebe/blomqvist#736]
Hej, mam problem z wtyczką Canal+VOD - od jakiegoś czasu strasznie laguje. Jestem przekonany, że miesiąc/dwa miesiące temu nie było takiego problemu. Nie wiem czy to moze być związane w wersją widevine? Buforuje stream średnio co pół sekundy. Ten problem nie występuje na Netflxie.

**Comment by Axis963** [comment]
Tak, to wina Widevine. Wejdź w InputStream Helpera, wybierz "Restore Widevine CDM library..." i wybierz "4.10.2252.5". Powinno pomóc.
